mne/preprocessing/tests/test_artifact_detection.py

```diff
@@ -0,0 +1,49 @@
+# Author: Adonay Nunes <adonay.s.nunes@gmail.com>
+#
+# License: BSD (3-clause)
+
+
+import os.path as op
+import warnings
+import numpy as np
+from numpy.testing import assert_allclose
+from mne.chpi import read_head_pos
+from mne.datasets import testing
+from mne.io import read_raw_fif
+from mne.preprocessing import (annotate_movement, compute_average_dev_head_t)
+
+data_path = testing.data_path(download=True)
+sss_path = op.join(data_path, 'SSS')
+pre = op.join(sss_path, 'test_move_anon_')
+raw_fname = pre + 'raw.fif'
+pos_fname = op.join(data_path, 'SSS', 'test_move_anon_raw.pos')
+
+with warnings.catch_warnings(record=True):
+    warnings.simplefilter("ignore")
+    raw = read_raw_fif(raw_fname, allow_maxshield=True).load_data()
```

Comment:
`allow_maxshield='yes'` then you don't need any `warnings` gymnastics (see [existing cHPI code](https://github.com/mne-tools/mne-python/blob/master/mne/tests/test_chpi.py#L51) for example)